src/layouts/MainLayout.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <q-layout view="hHh lpR fFf">
+    <q-header elevated>
+      <q-toolbar class="bg-grey-9">
+
+        <q-avatar square size="md">
+          <img alt="FlowLauncher Logo" src="/images/logo.svg">
+        </q-avatar>
+
+        <q-toolbar-title>
+          FlowLauncher
+        </q-toolbar-title>
+
+        <q-btn size="sm" flat stretch icon="mdi-github" type="a" href="https://github.com/Flow-Launcher/Flow.Launcher">
+          &nbsp;Flow-Launcher/Flow.Launcher
+        </q-btn>
+      </q-toolbar>
+      <q-toolbar class="bg-secondary shadow-8">
+        <q-tabs align="left" inline-label>
+          <q-tab icon="mdi-home" label="Start" @click="scrollToElement('home')" />
+          <q-tab icon="mdi-cogs" label="Features"  @click="scrollToElement('features')" />
+          <q-tab icon="mdi-download" label="Download"  @click="scrollToElement('download')" />
```

Comment:
I think it could add a **Descriptions tab** after **Features tab**.

Reply:
I included the Description directly under the start section. As the website is not that long, the auto scroll feature with the tabs isn't as effective as you see the content right in front of you even without scrolling. That's why I choose that the menu point Start will suffice for the description.